src/sentry/api/serializers/models/project.py

```diff
@@ -1020,6 +1023,17 @@ def serialize(
 
         return data
 
+    def get_custom_metric_cardinality_limit(self, attrs):
+        cardinalityLimits = attrs["options"].get("relay.cardinality-limiter.limits", [])
+        if not cardinalityLimits or len(cardinalityLimits) == 0:
+            return None
+
+        for limit in cardinalityLimits:
```

Comment:
You can simplify the code in this way:
```suggestion
        for limit in cardinalityLimits or ():
```